ballot/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.conf.urls import patterns, url
+from django.contrib import admin
+
+from . import views
+
+admin.autodiscover()
+
+urlpatterns = patterns(
+    '',
+    url(r'^measure/(?P<measure_id>[0-9]+)$', views.measure_view),
+    url(r'^ballot/$', views.ballot_view))
```

Comment:
@bcipolli yeah, the frontend URL doesn't make sense anyway. The frontend URL looks more like `city/12/elections`, so it's kind of implying the locality id would be used to fetch the ballot. That's not really true, since it's assuming there is a single ballot per city locality, but in reality it's one ballot per precint. We've made a call to gloss over this detail and only return a single ballot for the city locality, but I'm not really sure if that detail should be handled on the frontend or backend (fwiw, leaning toward the frontend).

> My inclination was to do this later, to make each endpoint "live" / more useful on each end. What do you think?

I'm not 100% sure what you meant but I think for this PR, we should just accept an Id, and we'll figure out what that Id is (ballot, city, or something else) later.